Client/Frontend/Widgets/SiteTableViewController.swift

```diff
@@ -134,12 +135,8 @@ class SiteTableViewController: UIViewController, UITableViewDelegate, UITableVie
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        view.addSubview(tableView)
-        tableView.snp.makeConstraints { make in
-            make.edges.equalTo(self.view)
-            return
-        }
+        
+        addTableView()
```

Comment:
```suggestion
        setupView()
```

Our standard naming is `setupView` for functions wherein we... setup the view. haha. Makes it easier to navigate various files when certain things are shared across them. :)